packages/fake-browser/src/apis/storage.test.ts

```diff
@@ -35,6 +35,58 @@ describe('Fake Storage API', () => {
 
     await fakeBrowser.storage.local.clear();
     expect(await fakeBrowser.storage.local.get()).toEqual({});
+
```

Comment:
Can we use an `it.each` instead of just adding the session and sync storage to this one test block?

https://vitest.dev/api/#test-each

(`it` is an alias for `test`, so you can do `it.each` as well)

Reply:
solved by fc434c0